appserver/web/web-glue/src/main/java/com/sun/enterprise/web/PEVirtualServer.java

```diff
@@ -50,11 +50,4 @@ public final class PEVirtualServer extends VirtualServer {
 
```

Comment:
Should we not just delete this class now?
It seems like an extension without changing anything from the parent?

Reply:
Right, I think this was just a run of "Delete unused members", didn't notice it.